schema/product-spec/0.0.3/data.json

```diff
@@ -372,11 +372,18 @@
     "type": {
       "description": "Product type",
       "enum": [
-        "complex",
-        "amplitude"
+        "CPX",
+        "AML",
+        "QLK",
+        "THM",
+        "CSI",
+        "VID",
+        "ORT",
+        "QKO",
+        "THO"
```

Comment:
Quickortho nor orthorectified thumbnail is not a product types per se, so we don't need to list them here